UnicornEngine/video/source/ModelMatrix.cpp

```diff
@@ -0,0 +1,40 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/ModelMatrix.hpp>
+
+#include <glm/gtc/matrix_transform.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+ModelMatrix::ModelMatrix() : model( glm::mat4( 1.0 ) ), m_position(0, 0, 0)
```

Comment:
This Class currently duplicates GLM functionality, do we really need it? Maybe a struct with direct calls to the glm functions would be more appropriate?

Reply:
Look at SetPosition and Translate

Reply:
@Warezovvv Do you really need to have this mandatory accumulative translation in there? I don't think so. You might as well give the right to choose to the user code, and pretty much nothing going to change.